src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -135,8 +135,9 @@
     <member name="List.GroupByFunction">
       <summary>Use a function to determine how list items should be grouped.</summary>
       <param name="list">list of values</param>
-      <param name="function"></param>
-      <search>group,function</search>
+      <param name="groupFunction">Function to group list</param>
+      <returns name="groupedList">Grouped list</returns>
```

Comment:
If this output port name replaces the `var[]..[]`, I think @Amoursol and I agreed that we should at least explain that the return type is `var[]..[]` and what that means in the output description. It could be something like:
```
type: var[]..[] (list of arbitrary rank/levels)
```

Reply:
addressed in 368caca